devtools/ferc1-etl-debug.ipynb

```diff
@@ -0,0 +1,420 @@
+   "id": "d594beab-7c93-47e3-acb4-b18ef6d3b16e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pick one table to transform\n",
+    "TRANSFORMER = psm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d030e249-3858-4650-ba2c-8ef62316db37",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "#### Transform Step-by-Step"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c5f3389-58c8-40db-b71a-e74a1d4c8c09",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "start = TRANSFORMER.transform_start(\n",
+    "    raw_dbf=ferc1_dbf_raw_dfs[TRANSFORMER.table_id.value],\n",
+    "    raw_xbrl_instant=ferc1_xbrl_raw_dfs[TRANSFORMER.table_id.value][\"instant\"],\n",
+    "    raw_xbrl_duration=ferc1_xbrl_raw_dfs[TRANSFORMER.table_id.value][\"duration\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9aa10b65-9200-42c4-8c02-745228b46b7f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "main = TRANSFORMER.transform_main(\n",
+    "    start\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4e1e202-a6b0-4fb9-b9cc-eccbc2e3b813",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "end = TRANSFORMER.transform_end(\n",
+    "    main\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6b2ad2b-57e3-49bd-ba83-8a68f6c06b17",
+   "metadata": {},
+   "source": [
+    "#### Transform All Steps Together"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "87e11c18-f6a1-49b0-92d5-b9ebc38fe920",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "full = TRANSFORMER.transform(\n",
+    "    raw_dbf=ferc1_dbf_raw_dfs[TRANSFORMER.table_id.value],\n",
+    "    raw_xbrl_instant=ferc1_xbrl_raw_dfs[TRANSFORMER.table_id.value][\"instant\"],\n",
+    "    raw_xbrl_duration=ferc1_xbrl_raw_dfs[TRANSFORMER.table_id.value][\"duration\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e795f853-a752-44a6-86ac-644baf291f9b",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Transform All Tables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08da1d39-6272-435f-8241-8b3879078393",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": true
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "transformed_tables = {}\n",
+    "\n",
+    "for transformer in transformers:\n",
+    "    transformed_tables[transformer.table_id.value] = transformer.transform(\n",
+    "        raw_dbf=ferc1_dbf_raw_dfs[transformer.table_id.value],\n",
+    "        raw_xbrl_instant=ferc1_xbrl_raw_dfs[transformer.table_id.value][\"instant\"],\n",
+    "        raw_xbrl_duration=ferc1_xbrl_raw_dfs[transformer.table_id.value][\"duration\"]\n",
+    "    )"
+   ]
```

Comment:
this is 100% a nit (and so sorry i rlly should setup the nice notebook PR commenter). This `Transform All Tables` section could just be done by running the main coordinating function of the module:

`pudl.transform.ferc1.transform`

Reply:
I'll mention that in the notebook! But I also think visually it's nice to see all the components in the notebook too.